src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +212,21 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        startUserCommandFeedbackLoop();
+    }
+
+	/**
+	 * Starts a user input, console output loop for the user
+	 * T2A3: Reduce SLAP of Method
```

Comment:
Comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).